Tools/Frame_params/Sub/bluerov2-3_5.params

```diff
@@ -218,11 +218,6 @@ FS_TEMP_ENABLE,0
 FS_TEMP_MAX,62
 FS_TERRAIN_ENAB,0
 GCS_PID_MASK,0
-GND_ALT_OFFSET,0.000000000000000000
-GND_EXT_BUS,1
```

Comment:
GND_EXT_BUS is 1 here, not -1. Shoun't that be preserved?

Reply:
sub overrides it